### DISCUSSION TITLE
问一下alist-linux-amd64 这个编译好的二进制版本的日志在哪

### DISCUSSION THREAD

**Discussion by Kikyo18** [AlistGo/alist#458] [Q&A]
因为alist有时候会自己关掉，我也搞不懂为啥会 自己停掉。想看一下日志。

**Comment by xhofe** [comment]
使用systemd守护进程的话，就是：`journalctl -u alist`

**Comment by z0z0r4** [comment]
如果不是用进程守护的怎么办....建议添加保存日志的功能

**Comment by z0z0r4** [comment]
@Xhofe

**Comment by Kikyo18** [comment]
是这样的老哥。我是用screen创建了一个单独的窗口给alist运行，但他会莫名其妙的自己停止运行，我就想看看日志。wo发现停止运行后，用screen切到进程里看，只有这些内容。alist已经停止运行了。

![QQ截图20220202193237](https://user-images.githubusercontent.com/57771987/152146031-0bb5ba3c-7038-4459-b3d1-4eb62b1cbdd8.png)

**Reply by xhofe** [reply]
screen可以搜一下怎么记录日志，比如：
> https://blog.csdn.net/lovemysea/article/details/78344114

**Reply by Kikyo18** [reply]
> screen可以搜一下怎么记录日志，比如：
> 
> > https://blog.csdn.net/lovemysea/article/details/78344114

我图里发的就是screen里记录的最后的日志了，接着alist就中断了。。。好像没看见什么报错，只是这样的话，没法向你提供什么有用的信息去分析为啥中断，。

**Reply by xhofe** [reply]
现在开始记录下次再停止就有日志了。
